docs/index.rst

```diff
@@ -2,19 +2,47 @@
 ops library API reference
 =========================
 
+The `ops` library is a Python framework for writing and testing Juju charms.
+
+For more about the Charm SDK, see https://juju.is/docs/sdk.
+
+The library provides:
+
+- :ref:`ops_main_entry_point` used to initialise and run your charm.
```

Comment:
I'd keep it as 

> `ops.main`, the entry point to  your charm, used to initialise and run your charm;

I'd also use semicolons (;) between the list items (the full stop feels a bit much).

Reply:
adding a comma after entrypoint; using semicolons.

not changing the "ops.main entry point" title though.

Reply:
In my opinion we should drop the punctuation at the end of the list items altogether. The semicolons just look really noisy in the [rendered docs](https://ops--1320.org.readthedocs.build/en/1320/). It's a bulleted list, these aren't complete sentences anyway.